lib/active_storage/railtie.rb

```diff
@@ -1,4 +1,5 @@
 require "rails/railtie"
```

Comment:
He just meant you can nix the require since the require of "rails/engine" will include rails/railtie